unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Tests/WorldsSubSectionComponentControllerTests.cs

```diff
@@ -0,0 +1,183 @@
+using Cysharp.Threading.Tasks;
+using DCL;
+using DCL.Helpers;
+using DCL.Social.Friends;
+using DCLServices.PlacesAPIService;
+using DCLServices.WorldsAPIService;
+using ExploreV2Analytics;
+using NSubstitute;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using UnityEngine;
+using MainScripts.DCL.Controllers.HotScenes;
+using System.Threading;
+
+public class WorldsSubSectionComponentControllerTests
+{
```

Comment:
if you want it to be EditMode test and included in CI, then you need to put attribute before the class [Category("EditModeCI")]